src/Targets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -31,21 +31,32 @@
       </PropertyPageSchema>
 
      <ProjectCapability Include="VB;Managed"/>
+  </ItemGroup>
+
+  <!-- Targets -->
+
+  <!-- Returns Vbc command-line arguments for the language service -->
+  <Target Name="CompileDesignTime"
+          Returns="@(_CompilerCommandLineArgs)"
+          DependsOnTargets="_CheckCompileDesignTimePrerequisite;Compile"
+          Condition="'$(IsCrossTargetingBuild)' != 'true'"
+        >
+
+    <!-- VB Framework projects needs core library references. In the case of Command line scenario, the vbc injects this reference. -->
+    <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+      <MSCorlibPath>$(FrameworkPathOverride)\mscorlib.dll</MSCorlibPath>
+      <MSVBPath>$(FrameworkPathOverride)\Microsoft.VisualBasic.dll</MSVBPath>
```

Comment:
Doesn't the old LS test for vbcore - the VBRuntime property - if that's set to Embed then we shouldn't be adding a reference.